src/elf/parser.rs

```diff
@@ -189,19 +220,25 @@ impl<'mmap> Cache<'mmap> {
 
     /// Retrieve the raw section data for the ELF section at index
     /// `idx`.
-    fn section_data(&self, idx: usize) -> Result<&'mmap [u8]> {
+    fn section_data_impl(&self, idx: usize) -> Result<(&'mmap Elf64_Shdr, &'mmap [u8])> {
```

Comment:
nit: why not section_data_raw, literally how it is described in the comment

Reply:
Sure, will rename.